### ISSUE TITLE
Giant didn't attack bandit

### ISSUE THREAD

**Issue by AggravatedSpoon** [zediious/tpfz-sprinkles#124]
Modlist Version: latest 
Modlist Profile: refined 

**Describe the Bug**
Giant didn't attach the bandit when wandering 

https://github.com/zediious/tpfz-sprinkles/assets/130263797/2bb7aeb3-fc3b-4a55-8740-f601efd519d7

**Comment by zediious** [comment]
I know that a lot of the giants will not attack unless you go near their mammoths. If the giant’s mammoth is far away or is not present for some reason, they might just not attack as they don’t have a trigger.

When it comes to what’s on here though, not sure about it. Giants always tend to send those that actually attack flying with me. Going to look around for giants to mess with.

**Comment by zediious** [comment]
Have found that there is indeed a “NoAggro” variant of the Giant, it seems the one I found falls under it. This specific giant is what is used for the wandering "non-camp" giants.

I have changed the behavior of this NoAggro giant to have a similar aggro radius, however I have reduced the aggro range considerably so they won’t get angy as quick as the camp giants but will still not tolerate an approach.

Can confirm the wandering giants with or without mammoths are now standoffish as expected, and will attack when coming close. This is going to produce a lot more situations where these giants attack,
